test/tests/physics/diffusion_fv.i

```diff
@@ -0,0 +1,92 @@
+[Mesh]
+  [cmg]
+    type = CartesianMeshGenerator
+    dim = 2
+    dx = '1 2'
+    dy = '2 1'
+    ix = '2 3'
+    iy = '3 2'
+    subdomain_id = '0 1
+                    1 0'
+  []
+  [split_boundaries]
+    type = BreakBoundaryOnSubdomainGenerator
+    input = cmg
+  []
+  allow_renumbering = false
+[]
+
+[Physics]
+  [Diffusion]
+    [FiniteVolume]
```

Comment:
Is there a good reason to have a nested block for the spatial discretization, instead of doing
```
[Physics]
  [DiffusionFV]
```
To me, what you have makes more sense only if you plan to have multiple discretizations in the same input file, which seems unlikely.

Reply:
At this point it s personal preferrence. 
```
[Physics]
  [DiffusionFV]
```
```
[Physics]
  [Diffusion/FV]
```
```
[Physics]
  [Diffusion]
    [FiniteVolume]
```

I think I got used to having `FV` but actually I think Daniel is right that spelling things out is better

Reply:
this
```
[Physics]
  [Diffusion]
    [FiniteVolume].
```
could be a prelude to allowing this
```
[Physics]
  [Diffusion]
    [Default]
      [fuel]
```
or simply this
```
[Physics]
  [Diffusion]
    [fuel]
```

Reply:
```
[Physics]
  [Diffusion]
    [FiniteVolume]
```
is fine. Maybe it's better for users. For devs, maybe it's just a little confusing sometimes to pair it with the class registered to that syntax.

Reply:
Although rarish, I could definitely imagine people mixing discretizations in a single input. For instance, in an FSI simulation, I may simulate the fluid with an FV discretization and so want heat transfer physics with FV in that subdomain, and then the solid with a CFEM discretization and so want a heat transfer physics with FE in that subdomain